eAbsentee/form/templates/confirmation.html

```diff
@@ -66,6 +66,10 @@ <h1> Application Confirmed </h1>
       <a href="https://www.facebook.com/voteabsenteeva">
         <i class="fa fa-facebook-square fa-2x"></i>
       </a>
+      &emsp;
+      <a href="instagram.com/voteabsenteeva">
```

Comment:
let's specify protocol etc to be consistent w/ FB icon
```suggestion
      <a href="https://www.instagram.com/voteabsenteeva">
```